### PR TITLE
TW 117 Logout

### PR DESCRIPTION
**Pull request by drminh2807**
# Issue
- #117 

# Demo
## Mobile: First login

https://github.com/linagora/twake-on-matrix/assets/12546908/c96024f4-f2e5-4326-90d6-cfed724eb826

## Mobile: Restart

https://github.com/linagora/twake-on-matrix/assets/12546908/9f228574-5f6c-4b21-a508-c89648fcd971

## Web

https://github.com/linagora/twake-on-matrix/assets/12546908/b7e61dd7-d589-4208-ac7f-a3ec489adc5b

### CODE REVIEW

**Review by sherlockvn — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

File: lib/widgets/matrix.dart [hunk 1]
```diff
@@ -74,8 +74,10 @@ class MatrixState extends State<Matrix> with WidgetsBindingObserver {
   Store store = Store();
   BuildContext? navigatorContext;
   HomeserverSummary? loginHomeserverSummary;
+  String? logoutSsoUrl;
```
Comment (hoangdat) [inline lib/widgets/matrix.dart:77]: why not `authUrl`?

File: lib/pages/connect/connect_page_mixin.dart [hunk 2]
```diff
@@ -74,6 +84,30 @@ mixin ConnectPageMixin {
...
+        callbackUrlScheme: urlScheme,
+      );
+    } else {
+      try {
+        final result = await FlutterWebAuth.authenticate(
+          url: url,
+          callbackUrlScheme: urlScheme,
+        );
+        Logs().d('tryLogoutSso::result: $result');
+      } catch (e) {
```
Comment (hoangdat) [inline lib/pages/connect/connect_page_mixin.dart:105]: What is the different?

**Comment by hoangdat** [comment]
please demo for the second access app (without login)

File: lib/widgets/matrix.dart (hunk 1 above)
Comment (drminh2807) [reply lib/widgets/matrix.dart]: changed

File: lib/pages/connect/connect_page_mixin.dart (hunk 2 above)
Comment (drminh2807) [reply lib/pages/connect/connect_page_mixin.dart]: fixed. web no need separate logic anymore

**Review by hoangdat — APPROVED** [review]

**Pull request merged**